### DISCUSSION TITLE
Luna themes!

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2631] [Show and tell]
And Zune/Royale too :) I'm unsure about the licensing here, but b00merang's/ElBullAzul's Windows-alikes were released under GPL 2.0 so...yeah.

Why do this? Nostalgia, mostly. Liking not-flat-and-low-contrast themes. Because it can be done. To prove a point. To enjoy LXQt's amazing theming flexibility.

![LunaBlue](https://github.com/user-attachments/assets/af6e0f6f-21fa-44b1-9b99-ff0fa7172f99)
![LunaEmbedded](https://github.com/user-attachments/assets/3a3f2cd8-2561-4875-8d7b-063979d21c5e)
![LunaGreen](https://github.com/user-attachments/assets/35580c8c-37d6-4a71-84bd-49ee281f5aba)
![LunaRoyale](https://github.com/user-attachments/assets/f683a890-5398-4c70-842a-c05a1269696c)
![LunaRoyaleNoir](https://github.com/user-attachments/assets/3258bbfd-3bbd-4e44-b37c-fc28e19506f3)
![LunaSilver](https://github.com/user-attachments/assets/512b3981-926a-46ac-a8b5-dcfdb40af2b8)
![LunaZune](https://github.com/user-attachments/assets/5096b935-3781-452a-a309-aebcb6ec1913)

**Comment by x0wllaar** [comment]
Do they work on Wayland? ~~Also, can you please share the themes somewhere? Maybe upload them to your own repo?~~ Found them, thank you!

Love the Royale theme!
